### PR TITLE
Implement BUILD_SET and SET_UPDATE

### PR DESCRIPTION
**Pull request by Ace314159**

### CODE REVIEW

File: ASTNode.h [hunk 1]
```diff
@@ -358,6 +358,18 @@ class ASTList : public ASTNode {
     value_t m_values;
 };
 
+class ASTSet : public ASTNode {
+public:
+    typedef std::set<PycRef<ASTNode>> value_t;
```
[the remark this answers is not in the document]
Comment (Ace314159) [reply ASTNode.h]: I chose to use `std::deque` so we get fast insertions to the front.

**Comment by Ace314159** [comment]
I also added code to print out `frozenset` in `pycdas`. I noticed this was missing while debugging.

**Comment by Ace314159** [comment]
@zrax I'd love another review, thanks!

**Review by zrax — APPROVED** [review]
Looks great, thanks!

**Pull request merged**